scripts/main.js

```diff
@@ -71,4 +71,51 @@ const departments = {
     }
 }
 
-console.log(departments);
\ No newline at end of file
+
+const result1 = prompt('Over welke afdeling wil je meer informatie? Kies uit: [marketing / sales / customer-service]').toLowerCase();
+
+
+switch (result1) {
+    case "marketing":
+        console.log(`${result1} is een leuke afdeling om te werken. Er werken op dit moment ${departments.marketing.numberOfEmployees} medewerkers. `)
+        break;
+    case "sales":
+        console.log(`${result1} is een leuke afdeling om te werken. Er werken op dit moment ${departments.sales.numberOfEmployees} medewerkers`);
+        break;
+    case "customer-service":
+        console.log(`${result1} is een leuke afdeling om te werken. Er werken op dit moment ${departments["customer-service"].numberOfEmployees} medewerkers`);
+        break;
```

Comment:
Je zou dit nog wat kunnen versimpelen en code dubbeling voorkomen
```suggestion
    case "marketing":
    case "sales":
    case "customer-service":
        console.log(`${result1} is een leuke afdeling om te werken. Er werken op dit moment ${departments[result1].numberOfEmployees} medewerkers`);
        break;
```